tests/ludwig/encoders/test_image_encoders.py

```diff
@@ -677,3 +679,45 @@ def test_tv_wide_resnet_encoder(
     inputs = torch.rand(2, *pretrained_model.input_shape)
     outputs = pretrained_model(inputs)
     assert outputs["encoder_output"].shape[1:] == pretrained_model.output_shape
+
+
+# test TorchVision encoders that were excluded from the earlier tests due to large memory requirements
+@pytest.mark.large_memory
```

Comment:
It might make sense to make use of the more general `@slow` test decorator. We now have a custom action that runs these tests on dedicated hardware following https://github.com/ludwig-ai/ludwig/pull/3029

@geoffreyangus can comment on whether it would be appropriate to use that decorator in this case.

Reply:
converted to @pytest.mark.slow decorator.